### DISCUSSION TITLE
[6 febbraio 2023] Calcolabilità

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#5] [Calcolabilità]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/a4b09bc8-6d35-4d86-8464-53dd1cd7c3c4)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/fc1eb3a9-3a82-49c4-8e50-ecd8937e8a5a)
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/05913a61-5c1d-43a2-8c69-d2689938adf9)

-------

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

**Comment by 18xChris** [comment]
Per quanto riguarda il secondo punto propongo la dimostrazione presente sugli appunti di [CasuFrost](https://github.com/CasuFrost/University_notes)
![image](https://github.com/user-attachments/assets/9d6916e9-f221-4d2b-b695-8f28e3307fe3)

**Comment by daromacarlo** [comment]
Che dite, funziona?

<img width="1694" height="514" alt="Immagine 2026-02-02 223942" src="https://github.com/user-attachments/assets/a6457ef6-ca1f-460d-957b-33fcb9d8d859" />
<img width="1696" height="809" alt="Immagine 2026-02-02 223957" src="https://github.com/user-attachments/assets/5fe2ac2e-80b1-4e90-a22d-cf299ae1521b" />
<img width="1695" height="66" alt="Immagine 2026-02-02 224016" src="https://github.com/user-attachments/assets/20170da0-c5af-468e-b512-28f5296e96f5" />

**Reply by daromacarlo** [reply]
**_Punto 2_**

<img width="1477" height="772" alt="immagine" src="https://github.com/user-attachments/assets/f23d6180-9212-4513-9966-1c6f000a2dce" />
